electron/app/recoil/selectors.ts

```diff
@@ -178,7 +178,8 @@ export const labelFilters = selector({
       const none = get(atoms.filterLabelIncludeNoConfidence(label));
       const include = get(atoms.filterIncludeLabels(label));
       filters[label] = (s) => {
-        const inRange = range[0] <= s.confidence && s.confidence <= range[1];
+        const inRange =
+          range[0] - 0.005 <= s.confidence && s.confidence <= range[1] + 0.005;
```

Comment:
What is the rationale here? This should probably be adjusted for numeric field range filters, as well.

I really want to clean up significant digit logic for these sliders. It just requires some time figuring out how to control the Material UI slider component. Or getting rid of it...

Reply:
I had an object with a random confidence (so ~10 decimal places), and the range was getting truncated to [0.52, 0.52] at some point, so the object was always getting filtered out (and I couldn't adjust the range because there was only one object).

There's likely a better place to do this, but I wasn't sure where - advice would be appreciated.